client/lib/signup/themes-data.js

```diff
@@ -45,12 +45,12 @@ export const themes = [
 		verticals: []
 	},
 	{
-		name: 'Dyad',
-		slug: 'dyad',
+		name: 'Dyad 2',
```

Comment:
As we continue to refresh themes in Signup, I think it might be better for our users (esp. new ones) to leave off the name's "version" number. The slug and demo_uri can keep the number to make things work properly.

I can open a ticket to clean up the labels of the other refreshed themes.

Ref: pNEWy-95y-p2